docs/topics/operator-overloading.md

```diff
@@ -1,11 +1,26 @@
 [//]: # (title: Operator overloading)
 
-Kotlin allows us to provide implementations for a predefined set of operators on our types. These operators have fixed 
-symbolic representation (like `+` or `*`) and fixed precedence. To implement an operator, provide a [member function](functions.md#member-functions)
-or an [extension function](extensions.md) with a fixed name, for the corresponding type, that means a left-hand side type for binary operations and argument type for unary ones.
-Functions that overload operators need to be marked with the `operator` modifier.
+Kotlin allows you to provide custom implementations for the predefined set of operators on your types. These operators have
```

Comment:
I have doubts about "your type". Does it necessary to specify exactly *whose* types?